### ISSUE TITLE
Deprecated Action Signature

### ISSUE THREAD

**Issue by YtvwlD** [gemnasium/toolbelt#39]
When I'm running `gemnasium autoupdate run`, it prints the following warning:

    DEPRECATED Action signature.  Must be `cli.ActionFunc`.  This is an error in the application.
    Please contact the distributor of this application if this is not you.
    See https://github.com/codegangsta/cli/blob/master/CHANGELOG.md#deprecated-cli-app-action-signature

The link is https://github.com/urfave/cli/blob/master/CHANGELOG.md#deprecated-cli-app-action-signature.

**Comment by bogdanRada** [comment]
any updates on this?

**Comment by gravis** [comment]
I will do that today, sorry for the delay :)

**Comment by gravis** [comment]
Could you please check https://github.com/gemnasium/toolbelt/releases/tag/0.2.8
Thanks :)

**Comment by bogdanRada** [comment]
I just tried that but i am getting this error:

```bash
dpkg: error processing archive gemnasium-toolbelt_0.2.8_amd64.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 7 package 'gemnasium-toolbelt':
 'Depends' field, invalid package name '${shlibs': must start with an alphanumeric character
Errors were encountered while processing:
 gemnasium-toolbelt_0.2.8_amd64.deb

```

**Comment by gravis** [comment]
Sorry for that, it's also broken on my side. I have released 0.2.9 to fix it: https://github.com/gemnasium/toolbelt/releases/tag/0.2.9
Thanks!

**Comment by YtvwlD** [comment]
Could you push that update to apt.gemnasium.com?
